tests/fips/openssl/openssl_fips_dhparam.pm

```diff
@@ -12,10 +12,26 @@ use strict;
 use warnings;
 use testapi;
 use utils;
+use version_utils qw(is_sle_micro);
+
+sub create_user {
+    my $user = $testapi::username;
+    my $password = $testapi::password;
+    if (script_run("getent passwd $user") != 0) {
+        assert_script_run "useradd -m $user";
+        assert_script_run "echo '$user:$password' | chpasswd";
+    }
+    # Make sure user has access to tty group
+    my $serial_group = script_output "stat -c %G /dev/$testapi::serialdev";
+    assert_script_run "grep '^${serial_group}:.*:${user}\$' /etc/group || (chown $user /dev/$testapi::serialdev && gpasswd -a $user $serial_group)";
```

Comment:
I wonder if we could move this somewhere else, but for now, looks good

Reply:
Yes, good idea, as it's also used somewhere else. Do you mind if I do this in another PR, as it would need to touch other files non related to this PR?

Reply:
Indeed, yesterday I started looking where it would be the place and the ticket for this, but completely forgot to do anything actionable, added a comment at: https://progress.opensuse.org/issues/117028#note-17